4-WebApp-your-API/4-1-MyOrg/README-use-keyvault-certificate.md

```diff
@@ -0,0 +1,77 @@
+# Web app using client certificates with Microsoft.Identity.Web
+
+There are different approaches for [using certificates with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates).
```

Comment:
Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates) provides various ways for a developer to use a certificate instead of a client secret to authenticate their apps with Azure AD

Reply:
updated